Block/Js.php

```diff
@@ -73,12 +73,18 @@ protected function getJsScriptsString()
     {
         $moduleVersionComment = sprintf('<!-- BreadCheckout Module Version: %s -->', $this->getModuleVersion());
 
-        $sentrySdkScript = '<script src="https://browser.sentry-cdn.com/5.4.3/bundle.min.js" crossorigin="anonymous"></script>';
+        $sentrySdkScript = '<script type="text/x-magento-init">
+                {
+                    "*": {
+                        "Bread_BreadCheckout/js/sentry/5.4.3/bundle.min": {}
```

Comment:
just a suggestion, maybe remove version folder and append version number to js script itself, like 5.4.3-bundle.min.js

Reply:
👍 yeah good idea